### PR TITLE
Changelog entry for v0.1.1

### PR DESCRIPTION
**Pull request by weiji14**
**Description of proposed changes**

A bugfix release to finally support Windows! Preview it live at https://pygmt-git-changelog-v011.gmt.now.sh/changes.html. For reference, the previous changelog is at #420.

Changelog made by following the instructions at https://github.com/GenericMappingTools/pygmt/blob/master/MAINTENANCE.md#updating-the-changelog. The changelog itself includes the following sections:

* Highlights
* Enhancements
* Documentation
* Bug fixes
* Maintenance
* List of Contributors (sorted by number of commits)

TODO:
- [x] Format Pull Request links using ``sed -i.bak -E 's$\(#([0-9]*)\)$(`#\1 <https://github.com/GenericMappingTools/pygmt/pull/\1>`__)$g' changes.rst``.
- [x] Add Zenodo DOI link once it's reserved.
- [ ] Add in new Pull Requests merged in for [v0.1.1](https://github.com/GenericMappingTools/pygmt/milestones/0.1.x)

Fixes #

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

### CODE REVIEW

File: doc/changes.rst [hunk 1]
```diff
@@ -1,6 +1,45 @@
...
+* Redirect www.pygmt.org to latest, instead of dev (#423)
+
+Bug Fixes:
+
+* Set GMT_COMPATIBILITY to 6 when pygmt session starts (#432)
+* Improve how PyGMT finds the GMT library (#440)
+
+Maintenance:
+
+* Finalize fixes on WIndows test suite for v0.1.1 (#441)
```
Comment (seisman) [inline doc/changes.rst:33]:
```suggestion
* Finalize fixes on Windows test suite for v0.1.1 (#441)
```

File: doc/changes.rst [hunk 2]
```diff
@@ -1,6 +1,45 @@
...
+* Cache test data on Azure Pipelines (#438)
+* Remove the variable GMT_SHAREDIR from CI settings (#437)
```
Comment (seisman) [inline doc/changes.rst:35]: PR #437 is a follow-up of #434. So perhaps no need to list #437 here.
```suggestion
```

File: doc/changes.rst [hunk 3]
```diff
@@ -1,6 +1,45 @@
...
+
+This release contains contributions from:
+
+* `Dongdong Tian <https://github.com/seisman>`__
+* `Wei Ji Leong <https://github.com/weiji14>`__
+
```
Comment (seisman) [inline doc/changes.rst:41]: Add a horizontal rule to better separate the changelog between different versions.
```suggestion

----
```

Sceenshot:

image

File: doc/changes.rst (hunk 1 above)
Comment (weiji14) [reply doc/changes.rst]: Good catch!

**Comment by seisman** [comment]
The vim command will also replace the #xxx in old releases. Better to use the sed command below:

```
sed -i.bak -E 's$\(#([0-9]*)\)$(`#\1 <https://github.com/GenericMappingTools/pygmt/pull/\1>`__)$g' changes.rst
```

**Comment by weiji14** [comment]
> The vim command above also replaces `#xxx` (again) in old releases. Better to use the sed command below:
> 
> ```
> sed -i.bak -E 's$\(#([0-9]*)\)$(`#\1 <https://github.com/GenericMappingTools/pygmt/pull/\1>`__)$g' changes.rst
> ```

Good one, much prefer `sed` to be honest. This should be pretty much done. Do you want to release today or wait for tomorrow?

**Comment by seisman** [comment]
> Do you want to release today or wait for tomorrow?

No need to wait another day. Let's do it today.

**Pull request merged**